aiconfig-docs/docs/gradio-workbook.md

```diff
@@ -0,0 +1,144 @@
+  </picture>
+</div>
+
+### Parameters
+
+Parameters are variables that you can define and pass into any cell within your workbook. You can set both **global parameters** and **local parameters**. For example, you can define parameters such as language or product in a cell like this with the handlebars syntax as shown below: `"Write a poem in {{language}} about a salesman selling {{product}}."`
+
+#### Global Parameters
+
+You can set global parameters for your entire workbook. Click on **Global Parameters {}** on the top of your workbook and set your global parameters that can be used in any cell of the workbook.
+
+<div align="center">
+  <picture>
+    <img alt="workbook" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/631f33ac-cf66-4e40-a7aa-c28d37b2ffad" width="800"/>
+  </picture>
+</div>
+
+#### Local Parameters
+
+You can set local parameters that only apply to the cell you are in. Click on the cell sidebar and **Local Parameters**. Note: Local parameters will override the global parameters if they have the same name.
+
+<div align="center">
+  <picture>
+    <img alt="workbook" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/0935ac7c-5e68-42f1-befe-e279b027cac8" width="800"/>
+  </picture>
+</div>
+
+### Chaining
+
+You can reference the output from one cell in another cell. This allows you to create complex, interconnected workflows within your workbook.
+
+The summarization prompt below uses the output of the story_gen prompt using handlebars syntax
+(e.g `{{story_gen.output}}`)
+
+![chaining](https://github.com/lastmile-ai/aiconfig/assets/81494782/be65e264-3734-422e-9967-35a44d77cbe7)
+
+### Download
+
+Click **Download** to download the `aiconfig` file that captures the current state of your space in a JSON file. Example Space and corresponding `aiconfig` file:
+
+- [Gradio Workbook Space](https://huggingface.co/spaces/lastmileai/ai_story_generator)
+- [my_app.aiconfig.json](https://huggingface.co/spaces/lastmileai/ai_story_generator/blob/main/story_gen.aiconfig.json)
+
+There are 2 purposes of the aiconfig file.
+
+1. **Lock in edits to your space.** Refreshing your space does not save changes. You need to upload the downloaded `aiconfig` file to your space repo to lock in the state. Make sure you name your aiconfig as `my_app.aiconfig.json` - it’s referenced in app.py.
+2. **Build apps inspired by your space.** HuggingFace spaces offer cool demos, but building an app with the tested models and prompts is challenging. However, by downloading the `aiconfig` file, you can use the prompts (and models) from your space in your code with the [AIConfig SDK](https://github.com/lastmile-ai/aiconfig).
+
+### Share Workbook
+
+Click **Share Workbook** to get a link to a read-only version of your space! Even space viewers can showcase your prompts and model outputs with the rest of the world.
```

Comment:
"Even space viewers can showcase your prompts and model outputs with the rest of the world." --> I think we can delete this, or make the first sectence "link to a read-only version of your space to share with the rest of the world!"